rest-api-spec/test/indices.exists/20_read_only_index.yaml

```diff
@@ -0,0 +1,30 @@
+---
+"Test indices.exists on a read only index":
+
+  - do:
+      indices.create:
+        index: test_index_ro
+
+  - do:
+      indices.put_settings:
+        index: test_index_ro
+        body:
+          index.blocks.read_only: true
+
+  - do:
+      indices.exists:
+        index: test_index_ro
+
+  - is_true: ''
```

Comment:
It's needed to check the result of the previous 'indices.exists' request.

From [here](https://github.com/elastic/elasticsearch/blob/master/src/test/java/org/elasticsearch/test/rest/client/RestResponse.java#L90):
```
 is_true: '' means the response had no body but the client returned true (caused by 200)
```